libs/elements/cards/src/lib/elewa-news-card/elewa-news-card.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+import { _highlightedNews as newsContent } from '@elewa-website/data/ui/content-text';
+import { NewsItem } from '@elewa-website/models/schema/ui/cards';
+
+@Component({
+  selector: 'elewa-website-elewa-news-card',
+  templateUrl: './elewa-news-card.component.html',
+  styleUrls: ['./elewa-news-card.component.scss'],
+})
+export class ElewaNewsCardComponent implements OnInit{
+  @Input() newsData!: NewsItem[];
+
+  ngOnInit() {
+    this.newsData = newsContent;
```

Comment:
Work with one item

Reply:
There is only one item in the news, so the component rendering the news should share data with the child component

Reply:
This is the child component. You should just show one item, which is what your component receives (input). The parent component to this is what will import the mockData and pass it here.